run-tests

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Is there a reason why you're removing this? (I've been running this after commits to ensure they don't regress different versions of Spark installed)